setup_stand_dev: 0.1
particle_count: 200
effective_particle_count: 150
eps: 0.5
max_range: 15
max_half_angle: 1.5
observe_dt: 0.0

yaw_unwrap_threshold: 4.0
measurement_covariance: [0.3, 0.0, 0.0, 0.05]
input_noise: [0.2, 0.0, 0.0, 0.0, 0.2, 0.0, 0.0, 0.0, 0.05]
